doc/_zoomin/ncs.tags.yml

```diff
@@ -94,24 +182,29 @@ mapping_topics:
                                                                            "development-kits"]
   - nrf/app_dev/device_guides/nrf91/thingy91_updating_fw_programmer.html: ["thingy91", "nrf9160",
                                                                            "prototyping-platforms"]
-  - nrf/app_dev/device_guides/nrf91/nrf91_building.html: ["nrf9160", "thingy91", "nrf9161",
-                                                          "nrf9151", "development-kits",
+  - nrf/app_dev/device_guides/nrf91/thingy91x_updating_fw_programmer.html: ["thingy91x", "nrf9151",
+                                                                            "prototyping-platforms"]
+  - nrf/app_dev/device_guides/nrf91/nrf91_building.html: ["nrf9160", "thingy91", "thingy91x",
+                                                          "nrf9161", "nrf9151", "development-kits",
                                                           "prototyping-platforms"]
-  - nrf/app_dev/device_guides/nrf91/nrf91_programming.html: ["nrf9160", "thingy91", "nrf9161",
-                                                             "nrf9151", "development-kits",
+  - nrf/app_dev/device_guides/nrf91/nrf91_programming.html: ["nrf9160", "thingy91", "thingy91x",
+                                                             "nrf9161", "nrf9151",
+                                                             "development-kits",
                                                              "prototyping-platforms"]
   - nrf/app_dev/device_guides/nrf91/nrf91_testing_at_client.html: ["nrf9160", "nrf9161", "nrf9151",
                                                                    "development-kits"]
-  - nrf/app_dev/device_guides/nrf91/nrf91_snippet.html: ["nrf9160", "thingy91", "nrf9161",
-                                                         "nrf9151", "nrf9131", "development-kits",
+  - nrf/app_dev/device_guides/nrf91/nrf91_snippet.html: ["nrf9160", "thingy91", "thingy91x",
+                                                         "nrf9161", "nrf9151", "nrf9131",
+                                                         "development-kits",
                                                          "prototyping-platforms"]
   - nrf/app_dev/device_guides/nrf91/nrf9160_external_flash.html: ["nrf9160", "development-kits"]
-  - nrf/app_dev/device_guides/nrf70/index.html: ["evaluation-kits"]
   - nrf/app_dev/device_guides/nrf70/*.html: ["nrf70-series", "nrf7002", "nrf7001", "nrf7000",
                                              "development-kits"]
+  - nrf/app_dev/device_guides/nrf70/index.html: ["evaluation-kits"]
   - nrf/app_dev/device_guides/nrf70/nrf7002ek_dev_guide.html: ["evaluation-kits"]
   - nrf/app_dev/device_guides/nrf70/constrained.html: ["kconfig"]
   - nrf/app_dev/device_guides/nrf54l/*.html: ["nrf54-series", "nrf54l15"]
```

Comment:
We would need entries for "nrf54l05", "nrf54l10", since all the subpages are applicable for these targets as well.
@annwoj - Could you confirm this?

Reply:
discussed online with Anna, and decided to keep this entry as is.